python/equistore-operations/equistore/operations/__init__.py

```diff
@@ -46,9 +46,10 @@
 from .lstsq import lstsq
 from .manipulate_dimension import (
     append_dimension,
+    insert_dimension,
+    permute_dimension,
```

Comment:
Just a minor thing. Should we name this `permute_dimensions` with an extra `s`. The function is permuting several dimensions and not only one. Against this renaming would be that it currently fits in the name space of the other functions `inser_dimension`, `remove_dimension` etc.

Reply:
that sounds good